cmd/lk/proto.go

```diff
@@ -44,6 +45,16 @@ type protoTypeValidator[T any] interface {
 	Validate() error
 }
 
+type paginatedType[T any] interface {
+	protoType[T]
+	GetPage() *livekit.Pagination
```

Comment:
We plan to use this for phone numbers 
https://github.com/livekit/protocol/blob/main/protobufs/livekit_models.proto#L31
Should we account for this as well ?

Reply:
Sure, I'll take a look